src/client/shader.cpp

```diff
@@ -368,6 +368,7 @@ ShaderSource::~ShaderSource()
 {
 	MutexAutoLock lock(m_shaderinfo_cache_mutex);
 
+#if IRRLICHT_VERSION_MT_REVISION >= 15
```

Comment:
We inlined Irrlicht now (it's now part of the MT codebase). No need for version checks.

Reply:
I wrote the code with a slightly older version of the engine, so this must have slipped through when I tried resolving all the conflicts, I’ll fix it :)